src/dpr/utils/urlHelper.ts

```diff
@@ -9,6 +9,7 @@ const createUrlForParameters = (
   if (updateQueryParams) {
     queryParams = {
       ...currentQueryParams,
+      selectedPage: '1',
```

Comment:
Will this set the page back to 1 when changing the columns or page too? Or is that avoided here?

Reply:
It doesn't in the test app. Paging and filtering and columns seem to work as intended. Might be worth testing this in the dev env